torch/random.py

```diff
@@ -19,14 +20,25 @@ def set_rng_state(new_state: torch.Tensor) -> None:
     default_generator.set_state(new_state)
 
 
-def get_rng_state() -> torch.Tensor:
+def get_rng_state(device: Union[int, str, torch.device] = "cpu") -> torch.Tensor:
```

Comment:
`Union[int, str, torch.device]` can be replaced by `torch.Device`, hope that this change can make CI green.